catalyst/contrib/nn/criterion/recsys.py

```diff
@@ -0,0 +1,132 @@
+# flake8: noqa
+import torch
+from torch import nn
+
+
+class Pointwise(nn.Module):
+    """
+    Pointwise approaches look at a single document at a time in the loss function. 
+    For a single documetns predict it relevance to the query in time. 
+    The score is independeent for the order of the docuemtns that are in the query's resluts
+
+    Input space: single document d1
+    Output space: scores or relevant classes
+    """
+    def __init__(self) -> None:
+        super().__init__()
+    
+    def forward(self):
+        pass
+
+class PairwiseLoss(nn.Module):
+    """
+    Pairwise approached looks at a pair ofd documents at a time in the loss function.
+    Given a pair of documents the algorithm try to come up with the optimal ordering 
+    For that pair and compare it with the ground truth. The goal for the ranker is to 
+    minimize the number of inversions in ranker. 
+
+    Input space: pairs of documents (d1, d2)
+    Output space: preferences (yes/no) for a given doc.pair
+    """
+    def __init__(self) -> None:
+        super().__init__()
+    
+    def forward(self):
```

Comment:
shall it be `def forward(self, positive_score, negative_score)` here, just to state the intent?